src/features/base-layout/components/header/dropdown/dropdown-wrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from 'react';
+import classNames from 'classnames/bind';
+import { usePositionDropdown } from '@/app/hooks';
+
+import styles from './dropdown-wrapper.module.scss';
+
+const cx = classNames.bind(styles);
+
+export interface DropdownWrapperProps {
+  onMouseLeave: () => void;
+  onMouseEnter: () => void;
+  isOpen: boolean;
+  children: ReactNode;
+}
+export const DropdownWrapper = ({
```

Comment:
```suggestion

export const DropdownWrapper = ({
```

Reply:
Corrected in [refactor: 256 - add emty space](https://github.com/rolling-scopes/site/pull/258/commits/8fb8bafb1b1a034b03995e8d942b19d992a31c4b)